github/Repository.pyi

```diff
@@ -477,6 +477,7 @@ class Repository(CompletableGithubObject):
         branch: Union[Branch, _NotSetType] = ...,
         event: Union[str, _NotSetType] = ...,
         status: Union[str, _NotSetType] = ...,
+        created: Union[datetime, _NotSetType] = ...,
```

Comment:
Should this be Union[str, _NotSetType] ? since the parameter created is a string as mentioned in GitHub Docs
https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#list-workflow-runs-for-a-repository

Reply:
I think we can use str or datetime, if using datetime, we can convert it to str internally 

Also need to keep pyi synced with py.